adafruit_display_text/label.py

```
@@ -122,6 +125,7 @@ def __init__(self, font, **kwargs) -> None:
         self._padding_left = kwargs.get("padding_left", 0)
         self._padding_right = kwargs.get("padding_right", 0)
         self.base_alignment = kwargs.get("base_alignment", False)
+        self.label_type = kwargs.get("label_direction", "LTR")
```

Inside of init we should validate that the user set a correct one of the direction strings. If they didn't we can raise an error that tells them the direction they chose was invalid.

---

Yes good point, I will add the verification of the label directions

---

Thanks @FoamyGuy for the recommendation. changes were introduced  see https://github.com/adafruit/Adafruit_CircuitPython_Display_Text/pull/135/commits/343ce75834d57bfaf74d28135b81cef23ae12050